fundamentals/math-primer.md

```diff
@@ -72,7 +72,7 @@ Coordinate planes help us solve problems where we need to consider points that h
 If you've seen certain kinds of bar graphs or line graphs, and there's a horizontal x-axis, and a vertical y-axis, this is the exact same concept.
 
 ![A cartesian coordinate plane. A cyan point is displayed as (2, 3) and a fuchsia point is displayed at (-4, 1).](../assets/fundamentals-recap_math-primer_coordinate-plane.png)  
```

Comment:
This image displayed fine, but I noticed the one above on line 34 is not displaying when I preview the page. Does it show for you?

Reply:
Getting an AWS error when I run `learn preview .` I'll merge and release and if there are issues with the image then I"ll revert the change.